stacks-signer/src/signerdb.rs

```diff
@@ -49,6 +93,15 @@ CREATE TABLE IF NOT EXISTS signer_states (
     encrypted_state BLOB NOT NULL
 )";
 
+const CREATE_OUTBOUND_MESSAGES_TABLE: &str = "
+CREATE TABLE IF NOT EXISTS outbound_messages (
+    insertion_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    reward_cycle INTEGER PRIMARY KEY,
```

Comment:
I'm not sure how this is going to work if you have multiple messages with the same reward cycle?  You'll get a `UNIQUE constraint failed` error from `rusqlite` if you try to insert multiple messages for the same reward cycle.  Is it the case that you'll only ever insert at most one message per reward cycle?

Reply:
In this table, `outbound_messages` is a slice of strings all concatenated into one blob. So `outbound_messages` is something like `msg_a,msg_b,msg_c`.